graphbolt/src/fused_csc_sampling_graph.cc

```diff
@@ -1152,22 +1171,22 @@ int64_t Pick(
     SamplerArgs<SamplerType::NEIGHBOR> args, PickedType* picked_data_ptr) {
   if (probs_or_mask.has_value()) {
     return NonUniformPick(
-        offset, num_neighbors, fanout, replace, options, probs_or_mask,
+        offset, num_neighbors, fanout, replace, options, probs_or_mask.value(),
```

Comment:
Is this a bug?

Reply:
It is not, when a function expects a tensor, but we pass a `torch::optional<torch::Tensor>`, it automatically calls the `.value()` function of the optional.

Reply:
I can take back the change if you so prefer.

Reply:
Sorry, I wrote this code weeks ago so I misremembered, it is due to changing the type of the probs_or_mask argument of NonUniformPick in the change above from an optional to a Tensor.